docs/development.md

```diff
@@ -0,0 +1,110 @@
+# Development
+
+Developing on Group Project XBlock v2 and running tests requires, at the minimum:
+
+* python>=2.7,<3.0
+* pip>=6.0
+* node.js>=0.10
+* npm>=1.3.10 (might work with older versions, but not checked)
+
+Other dependencies are installed via `pip install -r requirements/dev.txt` and `npm install`.
+
+Group Project XBlock v2 contains a Makefile to help with most common operations: 
+
+    * `make` to install everything, run tests, check code quality and collect coverage report
+    * `make test_fast` to run all tests without provisioning environment (i.e. env should be already provisioned)
+    * `make quality` to run all quality checks - fails fast, so does not always output all the problems. When fixing
+        quality violations make sure to run `make quality` again until clean pass.
+    * `make clean` - cleans tests and coverage results.  
```

Comment:
@e-kolpakov Same comment as above: The rendered Markdown for these items looks a bit weird. I think you can fix that by removing the leading spaces before the items.